src/CMakeLists.txt

```diff
@@ -176,7 +176,8 @@ if(BUILD_EXE)
   INCLUDE(CPack)
   
   #grpc build server and client target
-  list(APPEND CMAKE_PREFIX_PATH ${CMAKE_CURRENT_BINARY_DIR}+"/_deps/grpc-src/cmake")
+  set(GRPC_DIR \"${CMAKE_CURRENT_BINARY_DIR}/_deps/grpc-src/cmake\")
+  list(APPEND CMAKE_PREFIX_PATH GRPC_DIR)
```

Comment:
@SebastianSchildt @wenwenchenbosch if not working maybe remove CONFIG REQUIRED? To me it is not clear what CONFIG REQUIRED means. Maybe you could help? gRPCConfig.cmake.in exists but cmake searches for gRPCConfig.cmake only.

Reply:
I do not think that `list(APPEND CMAKE_PREFIX_PATH...) and `find_package(gRPC CONFIG REQUIRED)` is necessary, because `FetchContent_MakeAvailable(gRPC)` should already provide grpc als package and make the necessary libraries and executable avaliable:
-  libprotobuf
- grpc++
- grpc++_reflection
- $<TARGET_FILE:protoc>)
 - $<TARGET_FILE:grpc_cpp_plugin>)